src/app/collection-page/collection-form/collection-form.models.ts

```diff
@@ -34,6 +34,11 @@ export const collectionFormModels: DynamicFormControlModel[] = [
     name: 'dc.description.abstract',
     spellCheck: environment.form.spellCheck,
   }),
+  new DynamicTextAreaModel({
+    id: 'thumbnail',
+    name: 'dc.description.thumbnail',
```

Comment:
@DanGastardelli : This `dc.description.thumbnail` field isn't a valid Qualified Dublin Core field, so I would not recommend adding it under `dc.description.thumbnail`.  Additionally, this is metadata **about the Bitstream** (logo), but you appear to be storing this metadata **on the Collection**.  

It's not a great idea to store this on the Collection, as it means that this `dc.description.thumbnail` field would be shared as Collection metadata whenever someone accesses the Collection from the REST API (or other machine interfaces like OAI-PMH).

Ideally, it'd be better to store this metadata as simply `dc.description` (which already exists) on the **Bitstream** object itself.  That'd be much clearer & cleaner than creating a new metadata field to store the information on the Collection.

If it's not possible or easy to store this metadata on the Bitstream itself, then we could consider keeping this metadata on the Collection, but store it in a "hidden" field like `dspace.thumbnail.description`.  This would be better than creating an invalid `dc.description` field.  But, this field would need to be created though in the `dspace/config/registries/dspace-types.xml` on the backend.